### ISSUE TITLE
Handle CRAM files

### ISSUE THREAD

**Issue by AlistairNWard** [chmille4/bam.iobio.io#93]

**Comment by anderspitman** [comment]
@AlistairNWard do you have a good CRAM dataset I can use to start playing with this?

**Comment by AlistairNWard** [comment]
We should probably update the text prompts when this is working. E.g. on the landing page, the button text is "choose bam url", and this should probably become "choose bam/cram url". Similarly, when you click on that button, the text box that appears should be updated to BAM/CRAM  URL etc.

**Comment by anderspitman** [comment]
I agree. I've actually been thinking a bit about how we want the UX to work here. Is it reasonable to force the user to use .bam and .cram extensions on their files? If we enforce that, it makes a lot of things simpler. Otherwise we'll need to implement a bunch of logic deeper down to verify file types and whatnot. But I'm assuming people are going to want to use public datasets a lot and I'm not sure if URLs ever end in something other than the correct extension.

**Comment by AlistairNWard** [comment]
Yes, it's reasonable to enforce files to have those extensions.
